### ISSUE TITLE
ABRT crash report from Fedora

### ISSUE THREAD

**Issue by Ablu** [clintbellanger/flare-engine#1378]
Hello,

I received an automatic crash report via abrt:
```
[abrt] flare-engine: deallocate(): flare killed by SIGABRT
```
~~~
Truncated backtrace:
Thread no. 1 (10 frames)
 #6 deallocate at /usr/include/c++/4.9.1/ext/new_allocator.h:110
 #8 _M_deallocate at /usr/include/c++/4.9.1/bits/stl_vector.h:178
 #9 ~_Vector_base at /usr/include/c++/4.9.1/bits/stl_vector.h:160
 #10 ~vector at /usr/include/c++/4.9.1/bits/stl_vector.h:425
 #11 EffectManager::~EffectManager at /usr/src/debug/flare-engine.0.19/src/EffectManager.cpp:39
 #12 StatBlock::~StatBlock at /usr/src/debug/flare-engine.0.19/src/StatBlock.cpp:585
 #13 ~Event at /usr/src/debug/flare-engine.0.19/src/EventManager.h:78
 #14 destroy at /usr/include/c++/4.9.1/ext/new_allocator.h:133
 #16 std::vector<Event, std::allocator<Event> >::_M_erase at /usr/include/c++/4.9.1/bits/vector.tcc:147
 #17 erase at /usr/include/c++/4.9.1/bits/stl_vector.h:1150
~~~

bugzilla bug: https://bugzilla.redhat.com/show_bug.cgi?id=1210866

Regards,

**Comment by dorkster** [comment]
This appears to be the same as #1197, which was fixed in 24180fe4b6758591afbb34346ade5e917379a4ba.

**Comment by Ablu** [comment]
Thanks for the info!
